docs/user/troubleshooting-guides/03-60-apirule-conversion-webhook-cert-issue.md

```diff
@@ -0,0 +1,26 @@
+# APIRule Conversion Webhook Not Working Because of Certificate Issue
+
+## Symptoms
+
+- There is an error about the conversion webhook certificate when fetching or creating an `APIRule` resource, for example:
+```bash
+Error from server: conversion webhook for gateway.kyma-project.io/v1beta1, Kind=APIRule failed: Post "https://api-gateway-webhook-service.kyma-system.svc:9443/convert?timeout=30s": x509: certificate has expired or is not yet valid
+```
+
+
+- There is an error in the `api-gateway-controller-manager` logs with the message `"Secret \"api-gateway-webhook-certificate\" not found"`, for example:    
+```bash
+ERROR	Reconciler error	{"controller": "secret", "controllerGroup": "", "controllerKind": "Secret", "Secret": {"name":"api-gateway-webhook-certificate","namespace":"kyma-system"}, "namespace": "kyma-system", "name": "api-gateway-webhook-certificate", "reconcileID": "a808b99f-6db6-47f5-a82e-8176811238ac", "error": "Secret \"api-gateway-webhook-certificate\" not found"}
+```
```

Comment:
```suggestion
  ```bash
  ERROR	Reconciler error	{"controller": "secret", "controllerGroup": "", "controllerKind": "Secret", "Secret": {"name":"api-gateway-webhook-certificate","namespace":"kyma-system"}, "namespace": "kyma-system", "name": "api-gateway-webhook-certificate", "reconcileID": "a808b99f-6db6-47f5-a82e-8176811238ac", "error": "Secret \"api-gateway-webhook-certificate\" not found"}
  ```
```